### PR TITLE
completed nav tasks

### PR DESCRIPTION
**Pull request by WWDavidson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,4 @@
...
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+<!-- {/* <svg xmlns="http://www.w3.org/2000/svg" className="apt-logo" width="175" height="175" fill="currentColor" class="bi bi-building" viewBox="0 0 16 16">
+        <path fill="blue" fill-rule="evenodd" d="M14.763.075A.5.5 0 0 1 15 .5v15a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5V14h-1v1.5a.5.5 0 0 1-.5.5h-9a.5.5 0 0 1-.5-.5V10a.5.5 0 0 1 .342-.474L6 7.64V4.5a.5.5 0 0 1 .276-.447l8-4a.5.5 0 0 1 .487.022zM6 8.694 1 10.36V15h5V8.694zM7 15h2v-1.5a.5.5 0 0 1 .5-.5h2a.5.5 0 0 1 .5.5V15h2V1.309l-7 3.5V15z"/>
+        <path fill="gold" d="M2 11h1v1H2v-1zm2 0h1v1H4v-1zm-2 2h1v1H2v-1zm2 0h1v1H4v-1zm4-4h1v1H8V9zm2 0h1v1h-1V9zm-2 2h1v1H8v-1zm2 0h1v1h-1v-1zm2-2h1v1h-1V9zm0 2h1v1h-1v-1zM8 7h1v1H8V7zm2 0h1v1h-1V7zm2 0h1v1h-1V7zM8 5h1v1H8V5zm2 0h1v1h-1V5zm2 0h1v1h-1V5zm0-2h1v1h-1V3z"/>
+      </svg> */} -->
```
Comment (halebeau) [inline README.md:4]: Is this a image?

Comment (WWDavidson) [reply README.md:4]: It's an SVG

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -12,21 +27,40 @@ class App extends Component {
...
+
+              {/* <Route  path="/users/sign_up"  component={this.props.new_user_route}/> */}
+
+              {/* <Route path="/apartmentshow" component={ApartmentShow} />
+              <Route path="/apartmentedit" component={ApartmentEdit} />
+              <Route component={NotFound}/> */}
+            </Switch>
+          </Router>
+
+
```
Comment (halebeau) [inline app/javascript/components/App.js]: Whitespace here

File: README.md (hunk 1 above)
Comment (WWDavidson) [reply README.md:4]: Screen Shot 2022-02-10 at 5 16 58 PM

**Review by halebeau — CHANGES_REQUESTED** [review]
If u haven't already, cleanup the whitespaces all over and then Re-push

**Review by halebeau — APPROVED** [review]

**Review by halebeau — APPROVED** [review]

**Pull request merged**